src/containers/shared/components/Transaction/AccountDelete/Simple.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Not all transactions have been migrated over to this format - they shouldn't be created like this, it won't work. You can use a dictionary/list for the ones that aren't migrated over yet.

Reply:
can you please elaborate

Reply:
@ckniffen commented last time that this would be fine. I created components for all the transactions, do I need to delete them and again create a dictionary for the ones that don't exist?

I think that'll create problems because then whenever someone will update a Transaction type he/she has to manually go and remove that key from the dictionary.

Either we should use dictionary for everything now and then remove it when all the transaction types have been migrated or create simple components for the ones that don't exist.

That's what I think will be better.

Reply:
Sorry.  I maybe worded it wrong.  There could be entry for the index.tsx of just the transaction grouping but not the components.  That way it will fall through to the old code until they are refactored.  The way you have it here will cause all of the not yet refactored transaction types to not render the right stuff.

Reply:
Ahh okay,
So now what I am doing is importing the transaction types from index.tsx (which are present) and for the rest (that are not yet migrated) defining them in the Ledger - index.ts file.


Like this:

    const TransactionCategories = {
      ...transactionTypes.TransactionCategory, // these are imported ones
      AccountDelete: 'Account', // these are which are yet to be modified
      AccountSet: 'Account',
      CheckCancel: 'Payment',
      CheckCash: 'Payment',
      CheckCreate: 'Payment',
      DepositPreauth: 'Account',
    }

Is this fine? @ckniffen @mvadari

Reply:
An example of how we've done it previously is here: https://github.com/ripple/explorer/blob/master/src/containers/Transactions/Simple/index.js#L27-L34

Reply:
Thank you @mvadari that helped! I've changed the code.